Documentation/kubernetes/install.rst

```diff
@@ -286,44 +286,174 @@ Deploying the DaemonSet
     .. parsed-literal::
 
       $ wget \ |SCM_WEB|\/examples/kubernetes/1.7/cilium.yaml
-      $ vim cilium.yaml
-      [adjust the etcd address]
 
   .. group-tab:: K8s 1.8
 
     .. parsed-literal::
 
       $ wget \ |SCM_WEB|\/examples/kubernetes/1.8/cilium.yaml
-      $ vim cilium.yaml
-      [adjust the etcd address]
 
   .. group-tab:: K8s 1.9
 
     .. parsed-literal::
 
       $ wget \ |SCM_WEB|\/examples/kubernetes/1.9/cilium.yaml
-      $ vim cilium.yaml
-      [adjust the etcd address]
 
   .. group-tab:: K8s 1.10
 
     .. parsed-literal::
 
       $ wget \ |SCM_WEB|\/examples/kubernetes/1.10/cilium.yaml
-      $ vim cilium.yaml
-      [adjust the etcd address]
 
   .. group-tab:: K8s 1.11
 
     .. parsed-literal::
 
       $ wget \ |SCM_WEB|\/examples/kubernetes/1.11/cilium.yaml
-      $ vim cilium.yaml
-      [adjust the etcd address]
 
+Adjusting the ConfigMap
+-----------------------
 
-After configuring the ``cilium`` `ConfigMap` it is time to deploy it using
-``kubectl``:
+After downloading the ``cilium.yaml`` file, open it with your text editor and
+change the `ConfigMap` inside of it with the necessary changes based on the
+following instructions.
+
+Adjusting etcd-config
+~~~~~~~~~~~~~~~~~~~~~
+
+First, make sure the ``etcd-config`` endpoints' have the correct addresses of
+your etcd nodes.
+
+If you are running more than one nodes simply add it to the list of endpoints.
+The list of endpoints can accept both domain names or IP addresses.
+Make sure you specify the correct port used in your etcd node.
+
+If etcd is running with TLS, there are a couple of changes that you need to do.
```

Comment:
Can we link to https://coreos.com/etcd/docs/latest/op-guide/security.html for more details?

Reply:
Done